src/components/Blog/CustomRichBlocks.tsx

```diff
@@ -22,19 +29,45 @@ interface ImageData {
   image: MediaAttributes;
 }
 
+interface WidgetRouteSettings {
```

Comment:
Duplication of https://github.com/lifinance/jumper.exchange/pull/910/files#diff-c1e6733c6c5412839aca9d7ae8f74d4e0739f922a1376a092498da06b3addc8cR6 woud be good to extract it

Reply:
Good idea! It´s refactored now